[New Entry] General: Virtual Reality (#1079)
Codecademy/docs: content/general/concepts/virtual-reality/virtual-reality.md

---
Title: 'Virtual Reality'
Description: 'Virtual Reality (VR) is the computer-generated simulation of a three-dimensional image or environment that can be interacted with in a seemingly real or physical way by a person using special electronic equipment, such as a helmet with a screen inside or gloves fitted with sensors'
Subjects:
  - 'Computer Science'
  - 'Web Development'
Tags:
  - 'Best Practices'
  - 'Memory'
CatalogContent:
  - 'paths/code-foundations'
  - 'paths/computer-science'
  - 'paths/front-end-engineer-career-path'
---

> Virtual reality (VR) is a technology that uses head-mounted headsets with displays to generate realistic images, sounds, and other sensations to put users into an immersive virtual environment. VR allows us to create unbounded worlds that people can walk around and interact with using their hands, to feel as if they were transported to another place.

Reviewer: ```suggestion
Virtual reality (VR) is a technology that uses head-mounted headsets with displays to generate realistic images, sounds, and other sensations to put users into an immersive virtual environment. VR can create unbounded worlds that people can walk around and interact with using their hands, to feel as if they were transported to another place.
```

Preference is for Docs to avoid direct address to the reader in the form of using "you", "we", or "us."

> It’s still considered the wild west of the tech industry⁠—fun, exciting, bustling of opportunities and things are looking very promising.

Reviewer: ```suggestion
It’s still considered the wild west of the tech industry⁠— fun, exciting, bustling of opportunities and things are looking very promising.
```

## Augmented Reality vs. Virtual Reality

> The difference between augmented reality (AR) and virtual reality (VR) is that augmented reality utilizes the current physical setting of the user while introducing a layer of synthetic digital, sonic or somatic presentation to enhance the user experience whereas VR involves use of a headset to facilitate an experience rooted in the graphic and sonic presentation within the VR setup as opposed to using the individuals more broad physical surroundings for interaction.

Reviewer: ```suggestion
The difference between augmented reality (AR) and virtual reality (VR) is that augmented reality utilizes the current physical setting of the user while introducing a layer of synthetic digital, sonic or somatic presentation to enhance the user experience whereas VR involves use of a headset to facilitate an experience rooted in the graphic and sonic presentation within the VR setup as opposed to using the individual's more broad physical surroundings for interaction.
```

> Augmented reality is poised to grow substantially in use during the 21st century as it has many potential uses that range from entertainment, medical, commercial, military, fashion and design etc.

Reviewer: ```suggestion
Virtual reality is poised to grow substantially in use during the 21st century as it has many potential uses that range from entertainment, medical, commercial, military, fashion and design etc.
```

## Virtual Reality Examples

> - [A-Frame](https://www.codecademy.com/learn/learn-a-frame): A-Frame is a popular open-source web framework for building virtual reality experiences in the web browser.

Reviewer: I'd like to see more specific examples like this that could be linked out to.

- Automotive industry: experience with build of a vehicle before commissioning expensive prototypes
- Healthcare: practice surgery on virtual bodies / VR used as pain relief for burn injuries.

> - Real estate:  explore houses online and save your time.

Reviewer: ```suggestion
- Real estate:  explore houses online and save time.
```

---
Title: 'Virtual Reality'
Description: 'Virtual Reality (VR) is the computer-generated simulation of a three-dimensional image or environment that can be interacted with in a seemingly real or physical way by a person using special electronic equipment, such as a helmet with a screen inside or gloves fitted with sensors'
Subjects:
  - 'Computer Science'
  - 'Web Development'
Tags:
  - 'Best Practices'
  - 'Memory'
CatalogContent:
  - 'paths/code-foundations'
  - 'paths/computer-science'
  - 'paths/front-end-engineer-career-path'
---

> Virtual reality (VR) is a technology that uses head-mounted headsets with displays to generate realistic images, sounds, and other sensations to put users into an immersive virtual environment. VR can create unbounded worlds that people can walk around and interact with using their hands, to feel as if they were transported to another place.

Reviewer 2: ```suggestion
Virtual reality (VR) is a technology that uses head-mounted headsets with displays to generate realistic images, sounds, and other sensations to put users into an immersive virtual environment. VR can create unbounded worlds that people can walk around and interact with using their hands, and to feel as if they were transported to another place.
```

> It’s still considered the wild west of the tech industry⁠— fun, exciting, bustling of opportunities and things are looking very promising.

Reviewer 2: ```suggestion
It’s still considered the wild west of the tech industry⁠ — fun, exciting, bustling of opportunities and things are looking very promising.
```

---

Virtual reality (VR) is a technology that uses head-mounted headsets with displays to generate realistic images, sounds, and other sensations to put users into an immersive virtual environment. VR can create unbounded worlds that people can walk around and interact with using their hands, and to feel as if they were transported to another place.

It’s still considered the wild west of the tech industry⁠— fun, exciting, bustling of opportunities and things are looking very promising.

## Augmented Reality vs. Virtual Reality

Augmented reality utilizes the current physical setting of the user while introducing a layer of synthetic digital, sonic or somatic presentation to enhance the user experience.
Whereas, VR involves use of a headset to facilitate an experience rooted in the graphic and sonic presentation within the VR setup as opposed to using the individual's more broad physical surroundings for interaction.

Virtual reality is poised to grow substantially in use during the 21st century as it has many potential uses that range from entertainment, wellbeing, education, tourism, military, art and design etc.

## Virtual Reality Examples

> - [A-Frame](https://www.codecademy.com/learn/learn-a-frame): A-Frame is a popular open-source web framework for building virtual reality experiences in the web browser.
> - [Automotive industry](https://techwireasia.com/2020/02/how-jaguar-land-rover-uses-vr-to-design-and-engineer-new-vehicles/): Experience with build of a vehicle before commissioning expensive prototypes
> - [Healthcare](https://www.ossovr.com/): Practice surgery on virtual bodies 
> - [BehaVR](https://www.behavr.com/): VR used as pain relief for burn injuries.
> - [Real estate](https://www.klapty.com/explore): Explore houses online and save time.
> - [Learning and development](https://engagevr.io): Learning experience in a risk-free space mostly for soft skills such as public speaking, and active listening.
> - [Tourism](https://globetrotter-vr.com/): Interactive virtual tours of places around the world.

Reviewer 3: ```suggestion
- Web development: [A-Frame](https://www.codecademy.com/learn/learn-a-frame) is a popular [open-source](https://www.codecademy.com/resources/docs/open-source) web framework for building virtual reality experiences in the web browser.
- Automotive industry: Companies like [Jaguar](https://techwireasia.com/2020/02/how-jaguar-land-rover-uses-vr-to-design-and-engineer-new-vehicles/) are incorporating VR into their vehicle design before commissioning expensive prototypes.
- Healthcare:
  - Softwares like [Osso VR](https://www.ossovr.com/) allow for practicing surgical techniques on virtual bodies.
  - [BehaVR](https://www.behavr.com/) is used as pain relief for burn injuries.
- Real estate: [Klapty](https://www.klapty.com/explore) features virtual tours of apartments and houses that are on the market.
- Education: Products like [Engagevr](https://engagevr.io) provides learning experiences in a risk-free space mostly for soft skills such as public speaking and active listening.
- Tourism: [Globetrotter](https://globetrotter-vr.com/) features interactive virtual tours of places around the world for users to consider which activities and destinations to try out.
```
